plugins/netskope/help.md

```diff
@@ -0,0 +1,565 @@
+    ],
+    "type": "exact",
+    "json_version": 2
+  },
+  "modify_by": "Netskope REST API",
+  "modify_time": "1997-01-01 00:00:00",
+  "modify_type": "Created",
+  "pending": 0
+}
+```
+
+#### Update a File Hash List
+
+This action updates the File Hash List with the values provided. This replaces the existing values with the new values, so include the existing values you want to keep in the list.
+
+##### Input
+
+|Name|Type|Default|Required|Description|Enum|Example|
+|----|----|-------|--------|-----------|----|-------|
+|list|[]string|None|True|Comma-separated list of file hashes (md5 or sha256), with the maximum size of the list being 8MB|None|["e28eb9739b6e84d0f796e3acc0f5b71e", "e28eb9739b6e84d0f697e3acc0f5b71a", "e28eb9839b6e74d0f696e3acc0f6b710"]|
+|name|string|None|True|Name of an existing File Hash List shown in the Netskope UI on the File Hash List page|None|ExampleExistingFileHashList|
+
+Example input:
+
+```
+{
+  "list": [
+  	"e28eb9739b6e84d0f796e3acc0f5b71e",
+  	"e28eb9739b6e84d0f697e3acc0f5b71a",
+  	"e28eb9839b6e74d0f696e3acc0f6b710"
+  ],
+  "name": "ExampleExistingFileHashList"
+}
+```
+
+##### Output
+
+|Name|Type|Required|Description|
+|----|----|--------|-----------|
+|message|string|False|API response message|
+|status|string|False|API response status|
+
+Example output:
+
+```
+{
+  "status": "success",
+  "msg": "File Filter Profile updated successfully"
+}
+```
+
+#### Upload JSON File Configurations
+
+This action is used to upload multiple configurations via a JSON file.
+
+##### Input
+
+|Name|Type|Default|Required|Description|Enum|Example|
+|----|----|-------|--------|-----------|----|-------|
+|urllist|file|None|True|File that includes urllist in JSON format, which must have extension as *.json|None|{"items": [{"name": "string", "data": {"urls": ["https://example.com"], "type": "exact" }}]}|
+
+Example input:
+
+```
+{
+  "items": [
+    {
+      "name": "string",
+      "data": {
+        "urls": [
+          "https://example.com",
+          "https://example.com"
+        ],
+        "type": "exact"
+      }
+    }
+  ]
+}
+```
+
+##### Output
+
+|Name|Type|Required|Description|
+|----|----|--------|-----------|
+|uploaded_urllist|[]object|False|Contains list of uploaded urllist objects|
+
+Example output:
+
+```
+[
+  {
+    "id": 1,
+    "name": "string",
+    "data": {
+      "urls": [
+        "https://example.com"
+      ],
+      "type": "exact",
+      "json_version": 2
+    },
+    "modify_by": "Netskope REST API",
+    "modify_time": "2022-02-01T12:20:23.000Z",
+    "modify_type": "Created",
+    "pending": 1
+  }
+]
+```
+
+### Triggers
+
+_This plugin does not contain any triggers._
+
+### Custom Output Types
+
+_This plugin does not contain any custom output types._
+
+## Troubleshooting
+
+_This plugin does not contain any troubleshooting information._
+
+# Version History
+
+* 1.0.0 - Initial plugin
+
+# Links
+
+## References
+
+* [Netskope](https://www.netskope.com/)
+
```

Comment:
Please add links to the Netskope API v1 and v2 documentation as well. I think [this](https://docs.netskope.com/en/rest-api-v2-overview-312207.html) is the proper link for v2 but if not please add the correct link.